downstream/modules/platform/proc-controller-configure-system.adoc

```diff
@@ -5,34 +5,29 @@
 The *System* tab enables you to complete the following actions:
 
 * Define the base URL for the {ControllerName} host
-* Configure alerts
+//* Configure alerts
 * Enable activity capturing
 * Control visibility of users
-* Enable certain {ControllerName} features and functionality through a license file
-* Configure logging aggregation options
+* Set {ControllerName} analytics settings
+//* Enable certain {ControllerName} features and functionality through a license file
+//* Configure logging aggregation options
 
 .Procedure
 
 . From the navigation panel, select {MenuAEAdminSettings}.
+. Click btn:[Edit].
 . Choose from the following *System* options:
-* *Miscellaneous System settings*: Enable activity streams, specify the default {ExecEnvShort}, define the base URL for the {ControllerName} host, enable {ControllerName} administration alerts, set user visibility, define analytics, specify usernames and passwords, and configure proxies.
-* *Miscellaneous Authentication settings*: Configure options associated with authentication methods (built-in or SSO), sessions (timeout, number of sessions logged in, tokens), and social authentication mapping.
-* *Logging settings*: Configure logging options based on the type you choose:
-+
-image::ag-configure-aap-system-logging-types.png[Logging settings]
-+
-For more information about each of the logging aggregation types, see the xref:assembly-controller-logging-aggregation[Logging and Aggregation] section.
+//* *Miscellaneous System settings*: Enable activity streams, specify the default {ExecEnvShort}, define the base URL for the {ControllerName} host, enable {ControllerName} administration alerts, set user visibility, define analytics, specify usernames and passwords, and configure proxies.
+//* *Miscellaneous Authentication settings*: Configure options associated with authentication methods (built-in or SSO), sessions (timeout, number of sessions logged in, tokens), and social authentication mapping.
+//* *Logging settings*: Configure logging options based on the type you choose:
+//+
```

Comment:
Does the "Choose from the following System options" line need to be removed, as the PR deletes the list of options?